app/views/course/assessment/submission/submissions/index.json.jbuilder

```diff
@@ -38,6 +39,13 @@ json.submissions @course_users do |course_user|
     json.dateSubmitted submission.submitted_at&.iso8601
     json.dateGraded submission.graded_at&.iso8601
     json.logCount submission.log_count
+    json.graders do
+      json.array! submission.grader_ids do |grader_id|
+        cu = course_users_hash[grader_id] || [0, 'Undefined']
```

Comment:
Can we have it as "Unknown" or just default to "System" instead?

Reply:
The purpose for this is when a course_user does not exist anymore. So i think we can use unknown instead.